design/oss_suite_release.md

```diff
@@ -0,0 +1,300 @@
+# Conjur OSS Suite Release
+
+## Feature Overview
+In this effort, we intend to build a formal release process for the _Conjur OSS Suite_,
+which we define as the set of components including the Conjur server, deployment utilities,
+command line interface (CLI), client libraries, and integrations with platforms and DevOps tools.
+
+At current, we have no formal process for releasing the full suite of tools in
+the Conjur OSS ecosystem so that:
+- It is clear to end users which version of Conjur OSS they should use, and what
+  integrations and tooling are compatible with that version.
+- Key use cases that involve more than two components working together are automatically
+  tested to ensure that the end-to-end experience in those use cases works as expected.
+- Development can continue on the master branch of individual components while
+  end users have a reliable way to pull "stable" releases for all components and
+  avoid unexpectedly running "edge" releases .
+- It is clear what features are officially released and which are still in progress -
+  at current, there is no way to easily track unreleased features in our broad suite
+  of tools and integrations, and formally creating a system for tracking unreleased
+  features will help us to do a better job of comprehensively testing and releasing
+  completed features.
+
+In this effort, we will build a formal Conjur OSS Suite Release process to begin
+addressing these concerns.
+
+## Key Characteristics of Conjur OSS Suite Release
```

Comment:
As we discussed, this could probably be changed to "Acceptance Criteria".  Also, we might want to remove some of the implementation details and move them down to the design section so that this focuses on "what" and not "how".

Reply:
we didn't talk about the implementation details here that you're suggesting we move - it would be good to touch base on this directly.